model/src/pyrenew/process/periodiceffect.py

```diff
@@ -110,9 +113,11 @@ def sample(self, duration: int, **kwargs):
         """
 
         return PeriodicEffectSample(
-            value=self.broadcaster(
-                data=self.quantity_to_broadcast.sample(**kwargs)[0],
-                n_timepoints=duration,
+            value=SampledValue(
+                self.broadcaster(
+                    data=self.quantity_to_broadcast.sample(**kwargs)[0].value,
+                    n_timepoints=duration,
```

Comment:
We definitely want these SampledValues be time-aware, I'd think. For now I'd use `self.t_start` and `self.t_unit`, but we may want functionality to optionally compute those at construction from a time-aware `quantity_to_broadcast`

Reply:
Created an issue to list all components that will need to be explicitly timeaware: #320. Added `PeriodicEffect` to the list. 

att @damonbayer